### PR TITLE
Refactor Add tests

### PR DESCRIPTION
**Pull request by jb3rndt**
Also fixes chats sorting

### CODE REVIEW

File: packages/TelegramClientTests-UI.package/TCTUChatsListTests.class/instance/testChatOrderUpdatesIfPositionChanges.st [hunk 1]
```diff
@@ -0,0 +1,9 @@
+testing
+testChatOrderUpdatesIfPositionChanges
+
+	| chat |
+	chat := self core chatsHandler chats last.
+	chat position: 0.
```
Comment (phkeese) [inline packages/TelegramClientTests-UI.package/TCTUChatsListTests.class/instance/testChatOrderUpdatesIfPositionChanges.st:6]: maybe access the last first one and use its position as a reference

File: packages/TelegramClient-Core.package/TCCChats.class/class/defaultSortBlock.st [hunk 2]
```diff
@@ -1,10 +1,10 @@
 instance creation
 defaultSortBlock
 
-	"sort chats of unknown position last"
+	"sort chats of unknown position last, higher position means more higher priority"
```
Comment (phkeese) [inline packages/TelegramClient-Core.package/TCCChats.class/class/defaultSortBlock.st:4]: `more higher` -> `higher`

**Review by phkeese — CHANGES_REQUESTED** [review]
just a few improvements, the rest looks good

File: packages/TelegramClientTests-UI.package/TCTUChatsListTests.class/instance/testChatOrderUpdatesIfPositionChanges.st (hunk 1 above)
Comment (jb3rndt) [reply packages/TelegramClientTests-UI.package/TCTUChatsListTests.class/instance/testChatOrderUpdatesIfPositionChanges.st:6]: The problem is that no chat has a position so I cant use any as a reference. Should we give every chat a position or just leave it this way?

Comment (phkeese) [reply packages/TelegramClientTests-UI.package/TCTUChatsListTests.class/instance/testChatOrderUpdatesIfPositionChanges.st:6]: maybe write a comment for that

**Pull request merged**